optimade/server/entry_collections/mongo.py

```diff
@@ -32,6 +32,8 @@
 
 
 class MongoCollection(EntryCollection):
+    """ Collection class for MongoDB backends, using pymongo. """
```

Comment:
You could copy-paste the doc string from `EntryCollection` here, updating `collection` to match the new types.